src/ChannelServer/Network/ChannelPacketHandler.cs

```diff
@@ -1150,6 +1151,60 @@ public void CZ_ITEM_SELL(ChannelConnection conn, Packet packet)
 		}
 
 		/// <summary>
+		/// Contains newly uncovered areas of a map.
+		/// </summary>
+		/// <param name="conn"></param>
+		/// <param name="packet"></param>
+		[PacketHandler(Op.CZ_MAP_REVEAL_INFO)]
+		public void CZ_MAP_REVEAL_INFO(ChannelConnection conn, Packet packet)
+		{
+			var mapId = packet.GetInt();
+			var visible = packet.GetBin(128);
+
+			var mapData = ChannelServer.Instance.Data.MapDb
```

Comment:
All indexed databases, like MapDb, have a Find method by default that searches for the given index in the most efficient way. Same as above, the TryGetValue the db uses is more efficient than a FirstOrDefault.